src/commands/app/get-url.js

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const chalk = require('chalk')
+
+const { flags } = require('@oclif/command')
+
+const BaseCommand = require('../../BaseCommand')
+const AppScripts = require('@adobe/aio-app-scripts')
+const { wrapError } = require('../../lib/app-helper')
+
+class GetUrlCommand extends BaseCommand {
+  async run () {
+    // cli input
+    const { args } = this.parse(GetUrlCommand)
+    const { flags } = this.parse(GetUrlCommand)
+    const scripts = AppScripts({ listeners: {} })
+
+    try {
+      const options = {}
+      options.action = args.action
+      options.cdn = flags.cdn
+      const urls = await scripts.getUrls(options)
+      if (urls.runtime) {
+        this.log('Runtime URLs')
+        Object.entries(urls.runtime).forEach(([key, value]) => {
+          this.log(chalk.blue(chalk.bold(`${key} `)) + ' - ' + chalk.blue(chalk.bold(`${value} `)))
+        })
+      }
+
+      if (urls.cdn) {
+        this.log('CDN URLs')
+        Object.entries(urls.cdn).forEach(([key, value]) => {
```

Comment:
Question: should we make the log JSON parseable ?

Reply:
is thing something we do for other commands o/o in general? If yes, then I can modify the PR. Else we can have a separate story to address all command o/p. WDYT?

Reply:
yes ok let's do this as part of another PR.
maybe we could have something like a global --json flag in the future

Reply:
looks like we have agreement. Resolving the conversation